source/jormungandr/jormungandr/authentication.py

```diff
@@ -144,7 +149,8 @@ def has_access(region, api, abort, user):
         # To manage database error of the following type we should fetch one more time from database
         # Can connect to database but at least one table/attribute is not accessible due to transaction problem
         if can_read_user():
-            abort_request(user=user)
+            context = 'User is undefined, but users are accessible in database'
```

Comment:
I'm not sure that I understand what this message means. Does the user exist or not?